other/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to the 'other' section of this repository will be documented
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+### 2024-08-30
+- Added af-plans/ApiScanExample.yaml
```

Comment:
Looks like I should have included this in a prev PR :/

Reply:
I thought it was :)